packages/components/tile/src/components/osds-tile/osds-tile.scss

```diff
@@ -7,7 +7,9 @@
 
 :host {
   box-sizing: border-box;
+  display: flex;
   flex-direction: row;
+  justify-content: space-between;
```

Comment:
hum not sure we should enforce this render by defaut, is that on the compo spec requirement?